docs/sources/enterprise/access-control/_index.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Fine-grained Access control"
+description = "Understand how to grant, change or revoke access to the Grafana resources"
+keywords = ["grafana", "fine-grained-access-control", "roles", "permissions", "enterprise"]
+weight = 100
++++
+
+# Fine-grained access control
```

Comment:
This is a bit tricky to do, as essentially everyone who has required permissions can manage access control. That said, when Grafana starts, in the beginning only Grafana Server Admin can do that, and Grafana Server Admin can later decide that someone else needs also to be able to do it and give them permissions.

I will look into where it would make sense to mention this somehow.